.github/workflows/main.yml

```diff
@@ -13,17 +13,17 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Set up Python 3.7
+      - name: Set up Python 3.9
```

Comment:
This doesn't impact end users at all. It's only the version we use to run our CI.